css/index.css

```diff
@@ -136,40 +150,105 @@ header .logo {
 .main-content {
    border-top: 2px solid black; 
    border-bottom: 2px solid black;
-   margin: 75px 0 35px 0;
-   padding: 35px 0;
+   padding: 35px 0px;
+   margin: 75px 3% 35px 3%;
    font-size: 16px;
    line-height: 18px;
 }
 
-.main-content .top-content {
-    display: flex;
-    justify-content: space-around;
-}
-
 .main-content .middle-img {
+    width: 100%;
     display: block;
     margin: 30px auto;
 }
 
+.main-content .top-content,
 .main-content .bottom-content {
     display: flex;
+    flex-direction: column;
     justify-content: space-around;
 }
 
-.main-content .bottom-content p {
-    padding-right: 20px;
+.main-content .text-content {
+    margin-top: 30px;
+}
+
+.main-content .text-content:first-child {
+    margin-top: 0px;
 }
 
 /* Contact Styles */
+.contact{
+    margin-left: 3%;
+}
 
 .contact p {
-    margin: 15px 0;
+    margin-top: 15px;
 }
 
 /* Footer Styles */
 
 footer {
     text-align: center;
     margin: 40px 0 20px;
+}
+
+@media (min-width: 768px) {
```

Comment:
Glad to see you implementing min-width and percentages. You are already ahead of the curve and hope you continue to do so!